src/Home/Home.tsx

```diff
@@ -1,42 +1,65 @@
 import React, { useState, useEffect } from 'react'
 import { homeCities } from '../helpers/cities'
 import { getHomeData } from '../helpers/dataFilter'
+import { getTestData } from '../helpers/apiCalls'
 import './Home.scss'
 import Card from '../Card/Card'
 import wildfire from '../assets/wildfire.jpg'
-import beach from '../assets/beach.jpg';
-import altitude from '../assets/altitude.jpg';
-import roadTrip from '../assets/roadTrip.jpeg';
+import beach from '../assets/beach.jpg'
+import altitude from '../assets/altitude.jpg'
+import roadTrip from '../assets/roadTrip.jpeg'
 import covid from '../assets/covid.png'
 
-const Home: React.FC = () => {
+interface HomeProps {
+	markLoaded: Function
+	initialLoad: boolean
+	homePageData: [string, {}[]] | undefined
+	getMatchDetails: Function
+	getAllDetailsData: Function
+}
+
+const Home: React.FC<HomeProps> = props => {
 	const [ cardList, setCardList ] = useState<Array<any>>([])
 
 	const cityDetails = async () => {
-		const getCityData = homeCities.map(async (city: {name: string, lat: number, long: number}) => {
-			const cityName = city.name
-			const data = await getHomeData(city.lat, city.long)
-			return [cityName, data]
-		})
-		const cityData = await Promise.all(getCityData)
-		makeCards(cityData)
+		if (props.homePageData) {
+			makeCards(props.homePageData)
+		} else {
+			const getCityData = homeCities.map(async (city: {locality: string, region: string, lat: number, long: number}) => {
+				const cityLocality = city.locality
+				const cityRegion = city.region
+				const cityLat = city.lat
+				const cityLong = city.long
+				const data = await getHomeData(city.lat, city.long)
```

Comment:
You can toggle this back to line 34 if you are working on styling so that the data isn't being fetched upon load.

Reply:
thank you!